sidecar/sidecar-script-alt.py

```diff
@@ -0,0 +1,90 @@
+import os
+import time
+import subprocess
+import requests
+import json
+import base64
+import sys
+from functools import reduce
+from decouple import config
+
+
+SOCKETFILE=os.getenv('SOCKETFILE') 
```

Comment:
Thank you for introducing the python script. Please comment on each significant step, so the new users better understand what we are doing here.